android/app/src/main/java/com/mattermost/share/RealPathUtil.java

```diff
@@ -172,4 +176,22 @@ public static String getMimeType(String filePath) {
         File file = new File(filePath);
         return getMimeType(file);
     }
+
+    public static void deleteTempFiles(final String tempFolder) {
+        try {
+            File dir = new File(tempFolder);
+            if (dir.isDirectory()) {
+                String[] files = dir.list();
+                for (int i =0; i < files.length; i++) {
+                    String file = files[i];
+                    if (file.startsWith("mmShare") || file.startsWith("tmp")) {
+                        File f = new File(dir, files[i]);
+                        f.delete();
```

Comment:
Worth recursing and just trying a `delete()` if `!isDirectory()` after the check on line 183?

Reply:
I don’t get what you mean here

Reply:
May not matter for the use case, but I was basically proposing a recursive deletion strategy for this folder vs. just deleting files at the root